rest-api/model/genomics.py

```diff
@@ -40,6 +40,7 @@ class GenomicSet(Base):
   # have mysql always update the modified data when the record is changed
   modified = Column('modified', DateTime, nullable=True)
 
+  version = Column('version', Integer, nullable=False, default=1)
```

Comment:
Before we can merge this pull request we need to determine if this is necessary. If so then I will add an alembic migration for it.

If I remove the `version` property I get:
```Traceback (most recent call last):
  File "/Users/jonathannetterville/vumc/raw-data-repository/rest-api/test/unit_test/genomic_test/genomic_validation_test.py", line 141, in test_withdrawn
    validate_and_update_genomic_set_by_id(genomic_set.id)
  File "/Users/jonathannetterville/vumc/raw-data-repository/rest-api/genomic/validation.py", line 42, in validate_and_update_genomic_set_by_id
    dao.member_dao.update(member)
  File "/Users/jonathannetterville/vumc/raw-data-repository/rest-api/dao/base_dao.py", line 438, in update
    return self.update_with_session(session, obj)
  File "/Users/jonathannetterville/vumc/raw-data-repository/rest-api/dao/base_dao.py", line 430, in update_with_session
    self._validate_update(session, obj, existing_obj)
  File "/Users/jonathannetterville/vumc/raw-data-repository/rest-api/dao/base_dao.py", line 413, in _validate_update
    if existing_obj.version != obj.version:
AttributeError: 'GenomicSetMember' object has no attribute 'version'
```

Reply:
I'm not sure about this part, any thought? @robabram

Reply:
I talked with @robabram about this and we agree that there is no concurrent write risk with the intended use case of these models so we can safely override `def _validate_update()` (in the same way that `ParticipantSummaryDao` does) to not require a `version` field.